zulipterminal/config/keys.py

```diff
@@ -487,6 +492,9 @@ def display_key_for_urwid_key(urwid_key: str) -> str:
     """
     Returns a displayable user-centric format of the urwid key.
     """
+    if urwid_key == " ":
```

Comment:
Did not add this as an entry to the `URWID_KEY_TO_DISPLAY_KEY_MAPPING` as we do not want to affect the behavior of space when it is used as a separator between words.